test/test_events.rb

```diff
@@ -152,7 +152,7 @@ def test_pid_formatter
   end
 
   def test_custom_log_formatter
```

Comment:
Oh, I didn't even realize this is a thing.

So what does this PR give us? I'm not sure the CustomFormatter class does anything, as it has the same interface as a Proc.

However, I think the config option is useful. I'm not sure how one would have added a custom formatter before 

I'm not sure there's any benefit to the CustomFormatter class added here.

Reply:
i remove the CustomFormatter class and used a proc